Gruntfile.coffee

```diff
@@ -117,4 +107,4 @@ This file is generated by `grunt build`, do not edit it by hand.
     json2 = pkg.jqueryJSON
     json1[key] = json2[key] for key of json2
     json1.author.name = pkg.author
-    grunt.file.write('chosen.jquery.json', JSON.stringify(json1, null, 2))
+    grunt.file.write('chosen.jquery.json', JSON.stringify(json1, null, 2) + "\n")
```

Comment:
Will deleting the package file remove it from the read-only list they have there? Should we even care?

I used the jQuery file for writing a lot of the Bower stuff in #2264. Removing it would need to be done in that PR (or after it lands).